static/src/javascripts/projects/common/modules/commercial/hosted-video.js

```diff
@@ -85,99 +85,103 @@ define([
     }
 
     function init() {
-        require(['bootstraps/enhanced/media/main'], function () {
```

Comment:
It was suspected that this promise was causing hosted problems, and was removed in https://github.com/guardian/frontend/pull/13616, so I've reinstated it too.